tools/terrabyte_ogc_api_processes/terrabyte_ogc_api_process_wrapper.xml

```diff
@@ -0,0 +1,49 @@
+<tool id="terrabyte_ogc_api_processes" name="Terrabyte OGC API Processes" version="0.1.0" profile="22.05">
+	<description>Water quality EO Application Package</description>
+	<requirements>
+		<requirement type="package" version="4.3.1">r-base</requirement>
+		<requirement type="package" version="1.20.4">r-getopt</requirement>
+		<requirement type="package" version="0.2.3">r-httr2</requirement>
+		<requirement type="package" version="1.8.7">r-jsonlite</requirement>
+	</requirements>
+	<command detect_errors="exit_code"><![CDATA[
+	Rscript '$__tool_directory__/terrabyte_ogc_api_process_wrapper.R'
+		--outputData '$output_data'
+]]></command>
+	<configfiles>
+		<inputs name="inputs" filename="inputs.json" data_style="paths"/>
+	</configfiles>
+	<inputs>
+		<param name="cookie" label="Cookie" optional="false" value="" type="text" help="See 'Help' for for more information regarding the cookie."/>
+		<param name="date" label="Date" optional="false" help="Center date for data query" type="text"/>
```

Comment:
Wasn't there a data-picker already? Or was that on our todo list?

How should a date look like? Which format?

Reply:
I am not aware of a date picker but added some explanation for the format.